### PR TITLE
Change attributes to properties and add mutable properties spec

### PR DESCRIPTION
**Pull request by Yuripetusko**

### CODE REVIEW

File: standards/rmrk2.0.0/interactions/setattribute.md [hunk 1]
```diff
@@ -4,72 +4,80 @@ Discussing here: https://app.clickup.com/6615036/docs/69vzw-1501/69vzw-121
...
+  }
+},
 ```
 
 We can change the color to red like so:
 
 ```txt
-rmrk::SETATTRIBUTE::2.0.0::5105000-0aff6865bed3a66b-DLEP-DL15-00000001::Hair%20color::red
-```
+RMRK::SETATTRIBUTE::2.0.0::5105000-0aff6865bed3a66b-DLEP-DL15-00000001::color::%7B%22type%22%3A%22string%22%2C%22value%22%3A%22red%22%7D
```
Comment (Swader) [inline standards/rmrk2.0.0/interactions/setattribute.md:38]: Why use type and value here when it's implied that you can only change value and that type will match? Should be doable in the `color::red` way, no?

File: standards/rmrk2.0.0/entities/collection.md [hunk 2]
```diff
@@ -35,37 +35,48 @@ A collection MUST adhere to the following standard.
...
-- Collection-inherited Attribute
-- NFT's own Attribute
+- Metadata Properties
+- Collection-inherited Properties
+- NFT's own Properties
 
 Due to on-chain storage concerns and constraints, this function will usually be reserved only for
 [mutable attributes](../interactions/setattribute.md).
 
 ### Attribute Format
```
Comment (Swader) [inline standards/rmrk2.0.0/entities/collection.md:58]: "Property Format" for consistency?

File: standards/rmrk2.0.0/entities/collection.md [hunk 3]
```diff
@@ -35,37 +35,48 @@ A collection MUST adhere to the following standard.
...
 
-- Metadata Attribute
-- Collection-inherited Attribute
-- NFT's own Attribute
+- Metadata Properties
+- Collection-inherited Properties
+- NFT's own Properties
 
 Due to on-chain storage concerns and constraints, this function will usually be reserved only for
 [mutable attributes](../interactions/setattribute.md).
```
Comment (Swader) [inline standards/rmrk2.0.0/entities/collection.md:56]: mutable properties and should we change `setattribute` to `setproperty` or?

**Review by Swader — CHANGES_REQUESTED** [review]

File: standards/rmrk2.0.0/interactions/setattribute.md (hunk 1 above)
Comment (Yuripetusko) [reply standards/rmrk2.0.0/interactions/setattribute.md:38]: what if you want to change a type too, so from int to string?

File: standards/rmrk2.0.0/entities/collection.md (hunk 3 above)
Comment (Yuripetusko) [reply standards/rmrk2.0.0/entities/collection.md:56]: Yeah I thought about it too, to be honest I hate Property, I liked attribute more. But I wanted to copy what Enjin is doing. maybe this was a mistake, but it's too late now

File: standards/rmrk2.0.0/interactions/setattribute.md (hunk 1 above)
Comment (Swader) [reply standards/rmrk2.0.0/interactions/setattribute.md:38]: I don't think that should be allowed

File: standards/rmrk2.0.0/entities/collection.md (hunk 3 above)
Comment (Yuripetusko) [reply standards/rmrk2.0.0/entities/collection.md:56]: Ok I will change SETATTRIBUTE to SETPROPERTY

File: standards/rmrk2.0.0/interactions/setattribute.md (hunk 1 above)
Comment (Yuripetusko) [reply standards/rmrk2.0.0/interactions/setattribute.md]: Alright, will edit consolidator and change this part in spec

**Review by Swader — APPROVED** [review]

**Pull request merged**